apps/mobile/src/navigation/HomeNavigator.js

```diff
@@ -57,6 +56,7 @@ export default function HomeNavigator () {
   return (
     <HomeTab.Navigator {...navigatorProps}>
       {/* WebView screens (may link/route internally) */}
+      <HomeTab.Screen name='All Topics' component={HyloWebView} />
```

Comment:
Thought I'd give HyloWebView a spin here

Reply:
For now  I've done away with having special components or Screen entries for WebViews that don't have any special handling, and instead just direct the route in linking/index to `${AUTH_ROOT_SCREEN_NAME}/Drawer/Tabs/Home Tab/Web View`. I'd do the same for this as it currently has no special handling.

See https://github.com/Hylozoic/hylo/blob/3cd1b22ab6e0f861237ea56713cd43a1b13c8ab2/apps/mobile/src/navigation/linking/index.js#L63 and https://github.com/Hylozoic/hylo/blob/3cd1b22ab6e0f861237ea56713cd43a1b13c8ab2/apps/mobile/src/navigation/linking/index.js#L116.